src/wallet/wallet.cpp

```diff
@@ -4303,7 +4303,7 @@ int CWallet::ScanForWalletTransactions(
 
```

Comment:
If `pindexStart` is null then we encounter the same bug. Could it ever be null? If it's null then we encounter the same bug.